src/Apps/Artist/Routes/Consign/Utils/getConsignmentData.ts

```diff
@@ -7,7 +7,11 @@ import { flatten, groupBy } from "lodash"
 import { staticCSVToJSONData } from "./staticCSVToJSONData"
 
 export interface ArtistConsignment {
-  artworks: object[]
+  artworks: Array<{
+    internalID: string
+    realizedPrice: string
```

Comment:
This is purely for code intellisense, in the sense that we reference aliased CSV column fields (which aren't too pretty)